src/EnergyPlus/Pumps.cc

```diff
@@ -1919,6 +1919,9 @@ void CalcPumps(EnergyPlusData &state, int const PumpNum, Real64 const FlowReques
         FracFullLoadPower = thisPump.PartLoadCoef[0] + thisPump.PartLoadCoef[1] * PartLoadRatio + thisPump.PartLoadCoef[2] * pow_2(PartLoadRatio) +
                             thisPump.PartLoadCoef[3] * pow_3(PartLoadRatio);
         daPumps->Power = (FullLoadPowerRatio * daPumps->NumPumpsFullLoad + FracFullLoadPower) * FullLoadPower;
+        if (thisPump.EMSPressureOverrideOn) {
+            VolFlowRate = PartLoadVolFlowRate;
+        }
```

Comment:
How does the EMS Pressure override value end up meaning the pump should operate at `PartLoadVolFlowRate`?

Reply:
@Myoldmopar I think the PartLoadVolFlowRate is the flow rate from the pump based on load. When the power is calculated, it should be flowrate * Pressure head / motor efficiency. In other words, the current flow rate should be used, instead of full load flow rate. Of course, when EMS is used, the override pressure head can be a constant or variable, which depends on user input. We may discuss this issue in the next technicalities call.

Reply:
OK.  So just use values that would have been calculated from an overridden pressure head.  This makes sense in general, and I am willing to accept it.  Anyone else have any comments here before this moves toward merge?